doc/examples/edges/plot_circular_elliptical_hough_transform.py

```diff
@@ -82,9 +82,11 @@
 # -------------------
 #
 # The algorithm takes two different points belonging to the ellipse. It
-# assumes that it is the main axis. A loop on all the other points determines
-# how much an ellipse passes to them. A good match corresponds to high
-# accumulator values.
+# assumes that it is the major axis. A loop on all the other points determines
```

Comment:
This may be a bit much for the gallery example.

Reply:
I am personally in favor of the proposed change. It is much more clear this way.

Reply:
I agree with @soupault. It might be difficult to be concise here (aww, my suggestions are going to make it worse), but explicit/clear is better than...